### DISCUSSION TITLE
API Limits

### DISCUSSION THREAD

**Discussion by denizhanoter** [OwO-Network/DLX#156] [Q&A]
I'm using this API in the interface I'm writing with Python, it's useful for now, but I have some questions. 

First of all, I aim to use this API to translate files with many lines such as game language files. 
Is there a character count limit or a transaction request limit when translating? If so, what is our limit?

**Comment by Floquer** [comment]
I've already translated 203,000 lines and haven't reached the limit. It seems to me that there is no limit
